BambooTracker/gui/mainwindow.cpp

```diff
@@ -41,7 +41,6 @@
 #include <QMimeData>
 #include <QProgressDialog>
 #include <QRect>
-#include <QDesktopWidget>
```

Comment:
`QDesktopWidget` is obsolete in Qt5 & was removed in Qt6. I didn't see anything that actually used this include so I just removed it & everything appears to still work fine. If this was intended for some incomplete code, refer to Qt6's docs: https://doc.qt.io/qt-6/widgets-changes-qt6.html#qdesktopwidget-and-qapplication-desktop